include/lsst/jointcal/FitterBase.h

```diff
@@ -99,9 +99,15 @@ class FitterBase {
      * Saves results to text files "tupleName-meas" and "tupleName-ref" for the
      * MeasuredStar and RefStar tuples, respectively.
      * This method is mostly useful for debugging: we will probably want to create a better persistence
-     * system for jointcal's internal representations in the future.
+     * system for jointcal's internal representations in the future (see DM-12446).
      */
-    virtual void saveResultTuples(std::string const &tupleName) const = 0;
+    virtual void saveResultTuples(std::string const &tupleName) const;
+
+    /// Produces a tuple containing residuals of measurement terms.
```

Comment:
Imperative voice for Short Summary of methods: https://developer.lsst.io/docs/cpp_docs.html#short-summary

Reply:
Your mentioning this reminded me that I really should rename these methods. I've done so, and updated the docstrings.